src/Orchard.Web/Modules/Orchard.Projections/Services/DraftFieldIndexService.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Linq;
+using Orchard.Projections.Models;
+
+namespace Orchard.Projections.Services {
+    public class DraftFieldIndexService : IDraftFieldIndexService {
```

Comment:
I understand this is to prevent a breaking change. But I'd prefer to see it merged into the existing service in the dev branch. Can you file an issue for this? I will assign it for 1.11.

Reply:
Ok, i will do it.

Reply:
opened issue #7925